### DISCUSSION TITLE
[Question] How to set font for dropdown option?

### DISCUSSION THREAD

**Discussion by ztarma** [flet-dev/flet#2475] [Q&A] [answered]
### Question

page.theme.font_family = "Kanit"
my experiments have shown that the above methods work perfectly everywhere except for dropdown options.
Does anyone know the solution?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by Shaderd00d** [comment, marked as answer]
Hi.

You can use the text_style property for dropdown. Example:

`text_style=ft.TextStyle(font_family="Kanit", size=11, color=ft.colors.RED)`

**Reply by ztarma** [reply]
thx for your answer!
In addition, str in option cannot be set directly by ft.TextStyle. example: ft.dropdown.Option("test string", text_style),

so..correct way:
ft.Dropdown(
        text_style=my_style,
        ft.dropdown.Option("...")
       ....)
